### ISSUE TITLE
Feedback: "Modules Execution Process" - Average Duration is always 0s for every module

### ISSUE THREAD

**Issue by Ezard** [janbarari/gradle-analytics-plugin#131]
We've been recording stats for a week or so now, and the report is generally very useful!

However, the "Average Duration" and "Average Parallel Duration" stats seem to always be 0s for every module, despite other stats in this section being calculated (see image)

![image](https://github.com/janbarari/gradle-analytics-plugin/assets/5840701/8accbfd2-7cf2-49e6-b4a8-447a36b2457b)

In our root build.gradle.kts file, our config looks like:
```kotlin
gradleAnalyticsPlugin {
    database {
        ci = mysql {
            host = System.getenv("GRADLE_ANALYTICS_DB_HOST").orEmpty()
            name = System.getenv("GRADLE_ANALYTICS_DB_NAME").orEmpty()
            user = System.getenv("GRADLE_ANALYTICS_DB_USER").orEmpty()
            password = System.getenv("GRADLE_ANALYTICS_DB_PASS").orEmpty()
        }
    }

    trackAllBranchesEnabled = true

    trackingTasks = setOf(
        "koverXmlReport",
        "assembleDebug",
        "assembleRelease",
        "bundleRelease",
    )
}
```

Is there something missing in our config that we need to add in order to get these numbers to populate please?

Thanks!

**Comment by janbarari** [comment]
Hey @Ezard 
Thank you for sending the feedback, I will check this out and update you here.

Best,
Mehdi

**Comment by ilkupi** [comment]
I've also having this issue for a long time now

<img width="500" alt="image" src="https://github.com/user-attachments/assets/b82a94fd-bd93-4d0b-8db6-8e79e5bbdf25">

**Comment by janbarari** [comment]
Hi
This issue fixed in version `1.0.3`.
